### ISSUE TITLE
Path Symbol Upside Down.

### ISSUE THREAD

**Issue by nobodyczcz** [bshen95/MAPF-benchmark-web#27]
It is confirmed that the symbol "d" indicates y - 1 or "u" indicates y + 1, with (0,0) as the top left corner of the map, is a miscommunication result.

Need a clear definition of the coordination system and symbols to reflect the visualisation.

**Comment by bshen95** [comment]
No, this is definitely not.  This is confirmed by Daniel at the beginning of the development, we spend one weeks on this. 
In my mind, at least for all kind of games,  up means y increase, down means y decrease, left means x decrease and right means x increase.   Don't see why this is not right.
